### DISCUSSION TITLE
`Invoke-PasswordPolicyManager` with drift json fails due ESXi password policy.

### DISCUSSION THREAD

**Discussion by dgr-en** [vmware/powershell-module-for-vmware-cloud-foundation-password-management#87] [Community Help] [answered]
Hello you,

If I give the `Invoke-PasswordPolicyManager` a drift JSON that does not explicitly contain the ESXi policy `retry=3 min=disabled,disabled,disabled,7,7,` then the script fails.

I suspect there is an ESXi policy check that is not necessary. I would like to check against my specifications and not be bound by the specifications of the module.

Best Regards
Daniel

**Comment by tenthirtyam** [comment, marked as answer]
Hi @dgr-en 👋,

Thanks for posting this item in the discussions. 

This sounds like it would be a nice opportunity for an [enhancement request](https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-password-management/issues/new?assignees=&labels=enhancement&projects=&template=enhancement.yml).

Please feel free to open the issue and provide the requested details. We'll review and prioritize a milestone.

As always, [contributions are welcomed](https://vmware.github.io/powershell-module-for-vmware-cloud-foundation-password-management/community/contributing/).

**Ryan Johnson**
Senior Staff Solutions Architect | Product Engineering @ VMware, Inc.

**Reply by tenthirtyam** [reply]
Moved to https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-password-management/issues/88.
